deepmd/tf/descriptor/se_atten.py

```diff
@@ -727,12 +739,9 @@ def _pass_filter(
                         tf.reshape(self.avg_looked_up, [-1, 1]), [1, self.ndescrpt]
                     ),
                 )
+                #  (nsamples, natoms, nnei)
```

Comment:
I suggest commenting with `nloc` or `nall` instead of `natoms`, considering `nloc != nall` in the simulation.